src/mahoji/commands/easter.ts

```diff
@@ -1,8 +1,32 @@
-import { Time } from 'e';
+import { Bank, LootTable } from 'oldschooljs';
 
 import { client } from '../..';
-import addSubTaskToActivityTask from '../../lib/util/addSubTaskToActivityTask';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import { newChatHeadImage } from '../../lib/util/chatHeadImage';
 import { OSBMahojiCommand } from '../lib/util';
+import { mahojiUserSettingsUpdate } from '../mahojiSettings';
+
+const UniqueTable = new LootTable().add('Chickaxe').add('Leia');
```

Comment:
Maybe add Chickaxe to similarItems for dragon pickaxe?
or maybe it gives you eggs [worthless] while mining lol. just a wacky idea.

maybe just as an easter egg for the holiday